test/basic.js

```diff
@@ -65,12 +64,13 @@ tape('create', function (t) {
     },
   }
 
-  const msg1 = bb.create(mainContent, mfKeys, mainKeys, null, 1, timestamp())
+  const msg1 = bb.create(mainContent, mfKeys, mainKeys, null, 1, 12345)
```

Comment:
Removed monotonic-timestamp usage because I want the inputs to be always deterministic, so we always get the same `msg1` and same signature.